### PR TITLE
Generalize img frame

### PR DESCRIPTION
**Pull request by wjramos**
Since `.img-frame` so longer entirely captures what this does, added `.media-frame` alias. Maybe it should just be `.frame`?

Will replace `.embed-responsive-item`

### CODE REVIEW

File: src/less/mixins/images.less [hunk 1]
```diff
@@ -55,66 +35,59 @@
...
-  image-rendering: optimizeSpeed;             // Legal fallback
-  image-rendering: -moz-crisp-edges;          // Firefox
-  image-rendering: -o-crisp-edges;            // Opera
-  image-rendering: -webkit-optimize-contrast; // Safari
-  image-rendering: optimize-contrast;         // CSS3 Proposed
-  image-rendering: crisp-edges;               // CSS4 Proposed
-  image-rendering: pixelated;                 // CSS4 Proposed
-  -ms-interpolation-mode: nearest-neighbor;   // IE8+
+.sharpen {
+  image-rendering: pixelated;
```
Comment (wjramos) [inline src/less/mixins/images.less:64]: Autoprefixer captures this

File: src/less/mixins/images.less [hunk 2]
```diff
@@ -129,79 +102,101 @@
...
+    .media-anchor( @aspect; @anchor );
   }
 }
 
 
 
 // Application Classes
+.media-frame,
+.responsive-embed,
+.img-responsive,
```
Comment (wjramos) [inline src/less/mixins/images.less]: Aliases for removed components

File: src/less/mixins/images.less [hunk 3]
```diff
@@ -129,79 +102,101 @@
...
 }
 
-.img-frame ( @aspect; @crop: true; @anchor ) when ( ispixel( @crop ) ){
+.media-frame ( @aspect; @crop: true; @anchor ) when ( ispixel( @crop ) ){
   > img,
-  > video {
+  > video,
+  > iframe,
+  > embed,
+  > object {
```
Comment (wjramos) [inline src/less/mixins/images.less]: If there's more that should be captured, let me know

**Comment by mhewson** [comment]
I like media-frame, though the docs need to be updated

**Comment by mhewson** [comment]
approve 

[Approved with PullApprove](https://pullapprove.com/rei/rei-cedar/pull-request/68/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=rei/rei-cedar)

File: src/less/definitions/utilities/responsive-embed.less [hunk 4]
```diff
@@ -1,35 +0,0 @@
...
-    left: 0;
-    bottom: 0;
-    height: 100%;
-    width: 100%;
-    border: 0;
-  }
-}
-
-// Modifier class for 16:9 aspect ratio
-.embed-responsive-16by9 {
```
Comment (wjramos) [inline src/less/definitions/utilities/responsive-embed.less:28]: Should use `.ratio-16-9` instead

File: src/less/definitions/utilities/responsive-embed.less [hunk 5]
```diff
@@ -1,35 +0,0 @@
-// Embeds responsive
-//
-// Credit: Nicolas Gallagher and SUIT CSS.
-
-.embed-responsive {
```
Comment (wjramos) [inline src/less/definitions/utilities/responsive-embed.less:5]: Should use `.media-frame`

File: src/less/mixins/images.less [hunk 6]
```diff
@@ -9,26 +9,6 @@
...
-// on any device.
-// Rules are directly applied to <iframe>, <embed>, <video>, and <object> elements;
-// optionally use anexplicit descendant class .embed-responsive-item when you want
-// to match the styling for other attributes.
-//
-// Pro-Tip! You don't need to include frameborder="0" in your <iframe>s as we override
-// that for you.
-//
-//----------------------------------------------------------------------------------------------------------------------
-.img-responsive(@display: block) {
```
Comment (wjramos) [inline src/less/mixins/images.less:27]: This may be a little too ambitious to remove **need confirmation**

**Pull request merged**